doc/client.md

```diff
@@ -11,7 +11,7 @@ There are 3 clients: `FileGetContents`, `Curl` and `MultiCurl`.
 ```php
 $request = new PSR7Request('GET', 'https://example.com');
 
-$client = new Buzz\Client\FileGetContents(new Psr17ResponseFactory(), ['allow_redirects' => true]);
+$client = new Buzz\Client\FileGetContents(['allow_redirects' => true], new Psr17Factory());
```

Comment:
This has changed in dev-master. It was correct before your fix.

Reply:
Ah ok perfect, sry must have missed it :)